app/views/submissions/index.html.haml

```diff
@@ -4,6 +4,7 @@
 .scroll-panel-body
   = simple_search_form_for @query do |f|
     = f.input :status_start, collection: Submission::STATUS_MAPPING.keys, include_blank: true
+    = f.input :english_title_cont, label: "Tytuł"
```

Comment:
O właśnie, tego mi brakowało! Tylko że jeśli jest podany tytuł polski to przy indeksowaniu artykułów wyświetli się wtedy tylko polski tytuł. Co jeśli ktoś będzie chciał znaleźć artykuł po angielskim tytule, jeśli ten artykuł będzie miał też tytuł polski i tylko on będzie się wyświetlał?